### PR TITLE
[pt-br] Localize index page to Portuguese (Brazil) 

### PR DESCRIPTION
**Pull request by edsoncelio**
# Changes
 
> This is the new version of the PR #4615

 - Create `/content/pt/_index.md`
 - Add the language config to `hugo.yaml`
 
---

## Preview

https://deploy-preview-4794--opentelemetry.netlify.app/pt/

Contributes to #4834

### CODE REVIEW

File: content/pt/_index.md
Comment (chalin) [inline content/pt/_index.md:8]: This commit is quite old. Please review this page carefully to ensure that it corresponds to the `en` page (you'll want to pay particular attention to link references that have changes from absolute to relative).

**Review by chalin — CHANGES_REQUESTED** [review]
See inline comment for required changes to the landing page.

The spell-check issues will go away once #4816 is merged. Later we can look into re-enabling spelling checks for `pt` pages.

I'll update soon after your next round of updates. Thanks!

File: content/pt/_index.md
Comment (edsoncelio) [reply content/pt/_index.md:8]: Thanks! I'll take a look right away

Comment (edsoncelio) [reply content/pt/_index.md]: @chalin,

 I've added the commit based on [these](https://opentelemetry.io/docs/contributing/localization/#adding-default_lang_commit-to-new-pages) docs and this output from the `scripts/check-i18n.sh -i`:
```bash
08799298a05d17a6a659821bfe1c9d994e4ceba9 - hash of 'main' at HEAD
```

Regarding the content, I reviewed it and it is up to date, but I didn't find any change related to the links (at `_index.md`), could you please share an example that changed?

Comment (chalin) [reply content/pt/_index.md]: Did you run that command to see what the detailed diff was for the index page? That would have helped you see what changed.

In particular, the links into `/docs` should all be page relative (that is, they should not start with a slash).

That is, this PR contains:

```markdown
- [Saiba mais](/docs/what-is-opentelemetry/)
- [Experimente a demonstração](/docs/demo/)
```

But the [main English page](https://github.com/open-telemetry/opentelemetry.io/blame/main/content/en/_index.md) now has the following link (notice that there is no leading slash):

```markdown
- [Learn more](docs/what-is-opentelemetry/)
- [Try the demo](docs/demo/)
```

Carefully review the rest of the file too.

Comment (edsoncelio) [reply content/pt/_index.md]: Thank you, done!! My fault :sweat_smile:

File: content/pt/_index.md
Comment (chalin) [inline content/pt/_index.md:2]: For consistency across languages:

```suggestion
title: OpenTelemetry
```

**Review by chalin — APPROVED** [review]
LGTM once another native `pt-BR` speaker can approve the homepage content.

See inline comment for minor nit.

I'm OOO next week, but I'm ok with this getting merged if you feel comfortable with that @theletterf.

File: content/pt/_index.md
Comment (EzzioMoreira) [inline content/pt/_index.md:60]:
```suggestion
Crie e colete dados de telemetria a partir de seus serviços e softwares, e depois
```

File: content/pt/_index.md
Comment (EzzioMoreira) [inline content/pt/_index.md:61]:
```suggestion
encaminhe-os para uma variedade de ferramentas de análise. {{% /blocks/feature %}}
```

**Comment by EzzioMoreira** [comment]
LGTM 💯

File: content/pt/_index.md
Comment (jpkrohling) [inline content/pt/_index.md:1]: I believe it should be pt-BR, not pt.

**Review by jpkrohling — COMMENTED** [review]
Looks awesome. The only thing I'd change is to name this translation "Portuguese (Brazil)".

File: content/pt/_index.md
Comment (edsoncelio) [reply content/pt/_index.md:1]: Hey Juraci!
This was requested by chalin at [the old PR](https://github.com/open-telemetry/opentelemetry.io/pull/4615#discussion_r1632035874). 

I don't see this as a big problem; we are already using the [language code for pt-br.](https://github.com/open-telemetry/opentelemetry.io/pull/4794/files#diff-5928f4025679c9eee397ca3795367f71f667ce1f25018b7f7ae7c574da1f2f08R25).

**Comment by edsoncelio** [comment]
> Looks awesome. The only thing I'd change is to name this translation "Portuguese (Brazil)".

Regarding this, I'm following the [results of this discussion](https://github.com/open-telemetry/opentelemetry.io/pull/4615#discussion_r1630687873) in the old PR - but in short: To maintain consistency with other languages, ​​we can use only "Portuguese."

**Comment by jpkrohling** [comment]
Sounds good, if this was discussed before, I'm fine sticking with the outcome of that discussion.

**Review by jpkrohling — APPROVED** [review]

**Comment by svrnm** [comment]
thanks everyone, the PR is merged and the page is public:

https://opentelemetry.io/pt/ :tada:

**Pull request merged**
